src/Button/index.jsx

```diff
@@ -4,40 +4,68 @@ import PropTypes from 'prop-types';
 
 import styles from './Button.scss';
 
-function Button(props) {
```

Comment:
I have removed this destructuring because props that are not used immediately in the render but by other auxiliary functions get passed to the HTML button component as part of other. For example, inputRef is now used in setRefs and not in the render, so to prevent it from being passed explicitly to the HTML button, it would need to be explicitly named so that is not part of other, which is conceptually kind of confusing, since it's not being used in the render.